server/accounts.go

```diff
@@ -256,12 +256,28 @@ func (a *Account) String() string {
 	return a.Name
 }
 
+func (a *Account) setTraceDest(dest string) {
```

Comment:
These need to be public for embedded servers no?

Reply:
But since we can't really configure much an account programmatically (refer to Ari's previous comment), I figure that we don't make it public at all, at least for now. There was still a need for the getter/setter internally, this is why I added them.

Reply:
Ari comment was:
```
Where I embed NATS I often wish I had more options to configure accounts programatically - very little one can do now and it can be awkward to do things with accounts.
```